File: 242-valid-anagram/valid-anagram.cs
public class Solution {
    public bool IsAnagram(string s, string t) {

        if(s.Length != t.Length)
            return false;

        int[] charCount = new int[26];

        for(var i = 0; i < s.Length; i++)
        {
            charCount[s[i] - 'a']++;
            charCount[t[i] - 'a']--;
        }
        
        for(int i = 0; i < charCount.Length; i++){
            if(charCount[i] != 0)
                return false;
        }
        
        return true;
    }
}

File: 242-valid-anagram/valid-anagram.go
func isAnagram(s string, t string) bool {
    if len(s) != len(t){
        return false
    }

    var charCount [26]int

    for i := 0; i < len(s); i++ {
        charCount[s[i] - 'a']++
        charCount[t[i] - 'a']--
    }

    for i := 0; i < len(charCount); i++ {
        if charCount[i] != 0{
            return false
        }
    }
    return true
}